FHICORC.Core/Services/Model/SmartHealthCardModel/Shc/SmartHealthCardModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Globalization;
+using FHICORC.Core.Services.Interface;
+using FHICORC.Core.Services.Model.Converter;
+using Newtonsoft.Json;
+
+namespace FHICORC.Core.Services.Model.SmartHealthCardModel.Shc
+{
+    public class SmartHealthCardModel : ITokenPayload
+    {
+
+        [JsonConstructor]
+        public SmartHealthCardModel(Uri Issuer, DateTime IssuanceDate, VerifiableCredential VerifiableCredential)
```

Comment:
This JsonConstructor does not have any logic beside the default. Is this custom constructor needed or can it be removed?